_episodes/08-putting-it-all-together.md

```diff
@@ -146,25 +146,27 @@ styles and the source codes that create them.
 
 ### `plt` pyplot versus object-based matplotlib
 
-Matplotlib integrates nicely with the Numpy package and can use Numpy arrays
+Matplotlib integrates nicely with the numpy package and can use numpy arrays
 as input of the available plot functions. Consider the following example data,
-created with Numpy:
+created with numpy by drawing samples from a normal distribution:
 
 ~~~
 import numpy
-x = numpy.linspace(0, 5, 10)
-y = x ** 2
+mu, sigma = 0, 0.1
+s = np.random.normal(mu, sigma, 1000)
+
 ~~~
 {: .language-python}
 
-To make a scatter plot of `x` and `y`, we can use the `plot` command directly:
+To make a histogram plot of our draws from the normal distribution, we can use the `plot`
+command directly:
 
 ~~~
-plt.plot(x, y, '-')
+plt.hist(s)
 ~~~
 {: .language-python}
 
-![Line plot of y versus x](../fig/08_line_plot.png)
+![Line plot of y versus x](../fig/08-normal-distribution.png)
```

Comment:
This is no longer a line plot, right?

Reply:
Another point: if `08_line_plot.png` is no longer used, it shall be removed in the PR or marked as _unused_ by putting it into _`unused`_ folder or something like that...

Reply:
I'm happy with removal, if it sounds OK to you.

Reply:
`Line plot of y versus x` is not correct